test/functional/api/cas/casadm.py

```diff
@@ -104,12 +106,16 @@ def stop_cache(cache_id: int, no_data_flush: bool = False, shortcut: bool = Fals
 
 def add_core(cache: Cache, core_dev: Device, core_id: int = None, shortcut: bool = False):
     _core_id = None if core_id is None else str(core_id)
+    if isinstance(core_dev, Lvm):
+        core_dev_path = resolve_to_by_id_link(core_dev.path)
```

Comment:
I would prefer to require a lvm path to be provided by id instead.

Reply:
LVM class improved - now the "path" is the by-id path and we don't need changes for "add_core" function at all